agent/04_agent_configure.sh

```diff
@@ -288,8 +372,12 @@ if [ ! -z "${MIRROR_IMAGES}" ]; then
 
 fi
 
-if [[ "${NUM_MASTERS}" > "1" ]]; then
-  set_api_and_ingress_vip
-fi
+ if [[ "${NUM_MASTERS}" > "1" ]]; then
+    set_api_and_ingress_vip
+  fi
 
-generate_cluster_manifests
+if [[ $NETWORKING_MODE == "DHCP" ]]; then
```

Comment:
I don't think using DHCP to differentiate between agent-config and ZTP manifests is entirely correct. It is also possible to use agent-config with static IP addresses. Also possible would be to ZTP manifests in DHCP mode by not providing nmstateconfigs or nmstateconfigs configured to use DHCP if that is an option. I think we can refactor it in a future update. It would be better to have a choice of which type of manifests to use: install+agent config or ZTP manifests. And then for each manifest type, have static and DHCP options.

Reply:
Agree, to refactor if we add more scenarios in dev-scripts as mentioned above.

Reply:
I'd prefer to refactor it now as it seems somehow misleading,even though I won't block on this. It'd seem more natural to have a sort of `export AGENT_MANIFESTS={$AGENT_MANIFESTS:-installer}` config var default so that by default we generate installer config files (alternatively, it could be se to `ztp`)